examples/messageSign_test.go

```diff
@@ -74,7 +74,7 @@ func checkSignature(address string, message string, signature string) error {
 }
 
 func signMessage(t *testing.T, senderSeedHex string, message string) (string, string, string) {
-	keyGenerator := signing.NewKeyGenerator(signingCryptoSuite)
+	keyGenerator := crypto.NewKeyGenerator(signingCryptoSuite)
```

Comment:
so we moved the keygenerators from the signing package, directly in the root directory. :+1: